Userland/Libraries/LibJS/Runtime/Date.cpp

```diff
@@ -32,15 +32,16 @@
 
 namespace JS {
 
-Date* Date::create(GlobalObject& global_object, Core::DateTime datetime, u16 milliseconds)
+Date* Date::create(GlobalObject& global_object, Core::DateTime datetime, u16 milliseconds, bool invalid_date)
```

Comment:
Let's call this the same as the member:

```suggestion
Date* Date::create(GlobalObject& global_object, Core::DateTime datetime, u16 milliseconds, bool is_invalid)
```